datasets/wikipedia/wikipedia.py

```diff
@@ -441,6 +441,8 @@ def _base_url(lang):
 
             # Use dictionary since testing mock always returns the same result.
         downloaded_files = dl_manager.download({"xml": xml_urls})
+        if not pipeline.is_local():
+            downloaded_files = dl_manager.ship_files_with_pipeline(downloaded_files, pipeline)
```

Comment:
Beam pipelines can't download files (and this is so sad). So we have to upload the files somewhere the pipeline have access. Here it ships the files to the temp dir used by the pipeline